libs/mep/mwpm-143159/timeline/timeline.css

```diff
@@ -0,0 +1,142 @@
+.timeline {
+  box-sizing: border-box;
+  padding: 0 0 var(--spacing-xl) 0;
+  width: var(--grid-container-width);
+  max-width: 968px;
+  margin: 0 auto;
+}
+
+.timeline .row {
+  display: grid;
+}
+
+.timeline h1,
+.timeline h2,
+.timeline h3,
+.timeline h4,
+.timeline h5,
+.timeline h6,
+.timeline p {
+  color: var(--text-color);
+  margin: 0;
+  font-size: var(--type-body-xs-size);
+  line-height: var(--type-body-xs-lh);
+}
+
+.timeline .row:nth-child(2)>div {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.timeline .row:nth-of-type(1)>.right>div+div {
+  text-align: right;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.timeline .row:nth-of-type(1)>.left>div>* {
+  max-width: 100px;
+  padding-right: var(--spacing-m);
+}
+
+.dark .timeline .row:nth-child(3)>div *,
```

Comment:
@markpadbe I'm confused.  Are you trying to make sure that the font color is always "black" even when the theme is dark?  Are you also always forcing a white bg color?

Reply:
@vgoodric, the rules for the font color in the first row and 3rd row are different. The 3rd row font color will always be black and the first row font color will toggle between black and white; white if the block is a dark variant or in a dark section, otherwise, it will be black.
 
![Screenshot 2024-04-02 at 11 34 16 AM](https://github.com/adobecom/milo/assets/124626043/b3b2512e-ba18-45a7-8bb1-161a59ea2e43)